lua/neotest-plenary/init.lua

```diff
@@ -86,13 +90,19 @@ function PlenaryNeotestAdapter.build_spec(args)
       table.insert(filters, 1, parent_pos.range[1])
     end
   end
-  local min_init
-  for _, path in ipairs(async.fn.glob(("**%sminimal_init*"):format(lib.files.sep), true, true)) do
-    min_init = path
-  end
+  local min_init = config.min_init
   if not min_init then
-    for _, path in ipairs(async.fn.glob(("test*%sinit.vim"):format(lib.files.sep), true, true)) do
-      min_init = path
+    local globs = {
+      ("**%testrc*"):format(lib.files.sep),
```

Comment:
@mrjones2014 I think there is a bug here as I'm now seeing an issue 
```
The coroutine failed with this message: 
...e/nvim/lazy/neotest-plenary/lua/neotest-plenary/init.lua:96: invalid option '%t' to 'format
```
I'm not sure what you originally intended but it's reading `%t` as a pattern

Reply:
Whoops, you're right. It's supposed to be `%s` to format the file separator in. Here's a new PR to fix it: https://github.com/nvim-neotest/neotest-plenary/pull/6